docs/archetypes/authoring-guide.md

```diff
@@ -0,0 +1,363 @@
+
+> JSON Schema definitions increases the learning curve but it is necessary to preserve consistency of the archetypes and the parameters they depend on for deployment.
+
+---
+
+## Telemetry
+
+This reference implementation is instrumented to track deployment telemetry per module through [customer usage attribution](https://docs.microsoft.com/azure/marketplace/azure-partner-customer-usage-attribution).  When a new archetype is developed, the telemetry settings must be updated to reference the tracking id.  Telemetry configuration is located at [`config/telemetry.json`](../../config/telemetry.json).
+
+To support per-module tracking, we've split each archetype to be tracked independently.  At the moment, a single tracking id is used for all modules and can be modified in the future when required.
+
+### Instructions
+
+1. Add new archetype name & value in `customerUsageAttribution.modules.archetypes` object.  The name represents the new archetype and the value is the tracking id.
+
+    ```json
+    {
+      "customerUsageAttribution": {
+        "enabled": true,
+        "modules": {
+          "managementGroups": "a83f6385-f514-415f-991b-2d9bd7aed658",
+          "policy": "a83f6385-f514-415f-991b-2d9bd7aed658",
+          "roles": "a83f6385-f514-415f-991b-2d9bd7aed658",
+          "logging": "a83f6385-f514-415f-991b-2d9bd7aed658",
+          "networking": {
+            "nvaFortinet": "a83f6385-f514-415f-991b-2d9bd7aed658",
+            "azureFirewall": "a83f6385-f514-415f-991b-2d9bd7aed658"
+          },
+          "archetypes": {
+            "genericSubscription": "a83f6385-f514-415f-991b-2d9bd7aed658",
+            "machineLearning": "a83f6385-f514-415f-991b-2d9bd7aed658",
+            "healthcare": "a83f6385-f514-415f-991b-2d9bd7aed658"
+          }
+        }
+      }
+    }
+    ```
+
+2. Include telemetry tracking deployment in `main.bicep`.  Replace `NEW_ARCHETYPE_NAME` with the name defined in step 1.
+
+```bicep
+// Telemetry - Azure customer usage attribution
+// Reference:  https://docs.microsoft.com/azure/marketplace/azure-partner-customer-usage-attribution
+
+var telemetry = json(loadTextContent('../../config/telemetry.json'))
+module telemetryCustomerUsageAttribution '../../azresources/telemetry/customer-usage-attribution-subscription.bicep' = if (telemetry.customerUsageAttribution.enabled) {
+  name: 'pid-${telemetry.customerUsageAttribution.modules.archetypes.NEW_ARCHETYPE_NAME}'
+}
+```
+
+---
+
+## Deployment Instructions
+
+> Use the [Onboarding Guide for Azure DevOps](../onboarding/ado.md) to configure the `subscription` pipeline.  This pipeline will deploy workload archetypes such as Machine Learning.
+
+Parameter files for archetype deployment are configured in [config/subscription](../../config/subscriptions) folder.  The folder hierarchy is comprised of the following elements, from this folder downward:
```

Comment:
What's the purpose of the config/subscription and all these steps. It's not clear

Reply:
Added a clarification sentence.